src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
```

Comment:
Consider using a constant for the daily rental cost instead of the literal `40` to make the code more readable and maintainable. For example, `const DAILY_RENTAL_COST = 40;`.